platform-core-module/src/main/java/com/softicar/platform/core/module/permission/assignment/AGModuleInstancePermissionAssignmentTable.java

```diff
@@ -40,8 +40,7 @@ public void customizeAttributeProperties(IEmfAttributeList<AGModuleInstancePermi
 		attributes//
 			.editAttribute(AGModuleInstancePermissionAssignment.USER)
 			.setImmutable(true)
-			.setPredicateMandatory(EmfPredicates.always())
-			.setInputFactory(ModuleInstancePermissionAssignmentUserInput::new);
+			.setPredicateMandatory(EmfPredicates.always());
```

Comment:
Shouldnt there be a `.addFilter(user -> !user.isSystemUser());` here?

Reply:
hmm, not sure if really necessary, but I can do that

Reply:
ok, added, also added to `AGRoleUser` for symmetry

Reply:
I don't see the point of suppressing the system user only in two distinct places. What about the remaining 5000 user inputs, particularly in the payload software? IMO, we either need a systematic solution to this - or none.

Reply:
not sure if this has anything to do with this PR? I just retained the status quo and extended it to `AGRoleUser` for symmetry...so?